python_agent/README.md

```
@@ -0,0 +1,148 @@
+
+### Messages from Python to Java
+
+1.  **Request Observation:**
+    -   Plain string: `command:get_observation`
+
+2.  **Execute Action:**
+    -   Plain string starting with `command:execute_action:` followed by a JSON payload.
+    -   Example: `command:execute_action:{"action_type": "walk_to", "parameters": {"x": 3200, "y": 3200, "plane": 0}}`
+
+    **Supported `action_type` values and their `parameters`:**
+    *   `type_string`:
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)